.travis.yml

```diff
@@ -4,13 +4,13 @@ services:
   - docker
 
 env:
-  - DOCKER_IMAGE=pangeo/pangeo-notebook:2019.04.01 
+  - DOCKER_IMAGE=pangeo/pangeo-notebook:2019.04.19
 
 before_install:
 - docker pull $DOCKER_IMAGE
 - docker run -d --name pangeo $DOCKER_IMAGE
```

Comment:
As a quick fix, if you change this line to the line @jhamman mentioned, it should start working right away. I’ll take a deeper look soon since this should continue working.